packages/openapi-ts/src/utils/write/services.ts

```diff
@@ -480,7 +480,14 @@ export const processService = (
       accessLevel: 'public',
       comment: toOperationComment(operation),
       isStatic: config.name === undefined && config.client !== 'angular',
-      name: operation.name,
+      name: config.services.methodNameBuilder
+        ? camelcase(
```

Comment:
Isn't it the responsibility of whoever is implementing this to ensure they return the correct casing?

Reply:
Yeah, you're probably right - i was just going for consistency with the current implementation which would have resulted in a camelCase method.